lib/modules/manager/gradle/artifacts.ts

```diff
@@ -142,28 +192,49 @@ export async function updateArtifacts({
       ],
     };
 
-    const subprojects = await getSubProjectList(cmd, execOptions);
-    cmd += ` ${subprojects
-      .map((project) => project + ':dependencies')
-      .map(quote)
-      .join(' ')}`;
+    const cmds = [];
+    let lockfileCmd = '';
+    if (lockFiles.length) {
+      const subprojects = await getSubProjectList(baseCmd, execOptions);
+      lockfileCmd = `${baseCmd} ${subprojects
```

Comment:
See previous suggested change

```suggestion
      let lockfileCmd = `${baseCmd} ${subprojects
```